tests/bud.bats

```diff
@@ -1894,6 +1902,21 @@ function _test_http() {
   expect_output --substring "COPY --from=build: no stage or image found with that name"
 }
 
+@test "bud with copy-from referencing the current stage and with --keep-ownership" {
+  _prefetch busybox
+  target=busybox-derived
+  run_buildah build $WITH_POLICY_JSON -t ${target} -f $BUDFILES/copy-from-keep-ownership/Dockerfile $BUDFILES/copy-from-keep-ownership
+  expect_output --substring "\-rw-r--r--    1 123      456"
+}
+
+@test "bud with copy-workdir and with --keep-ownership" {
+  _prefetch busybox
+  target=busybox-derived
+  chown 321:999 $BUDFILES/copy-from-keep-ownership/file2.txt
+  run_buildah build $WITH_POLICY_JSON -t ${target} -f $BUDFILES/copy-from-keep-ownership/Dockerfile2 $BUDFILES/copy-from-keep-ownership
+  expect_output --substring "\-rw-r--r--    1 321      999"
```

Comment:
Whitespace and substring checks are fragile. Also, this will break if the base image ever (for whatever reason) gains those uids/gids. I encourage you to change your containerfiles to `RUN stat -c "%u:%g"` instead, and do an exact (not substring) check.